vaadin-checkbox-flow-demo/src/main/java/com/vaadin/flow/component/checkbox/demo/CheckboxView.java

```diff
@@ -57,6 +64,80 @@ protected void initView() {
         configurationForRequired(); // Validation
         themeVariantsHorizontal();// Theme Variants
         styling(); // Styling
+        dataViewDemo();
+    }
+
+    private void dataViewDemo() {
+        final CheckboxGroup<String> checkboxGroup = new CheckboxGroup<>();
+
+        final List<String> items = new ArrayList<>();
+        items.add("Item1");
+        items.add("Item4");
+        items.add("Item2");
+        items.add("Item3");
+
+        ListDataProvider<String> dataProvider = DataProvider.ofCollection(items);
```

Comment:
Should not demo this API but instead use most convenient way of doing things, which is the varargs method

Reply:
The problem is `Arrays.asList()` creates its own implementation of `ArrayList` which has no add/remove operations, that's why i choose `.ofCollection()` here. But, maybe, I shouldn't demo the cases with add/remove methods calls with `ListDataProvider.getItems()`, because this is out of scope for DataView.

Reply:
Back to vararg method.